driver/bpf/configure/RSS_STAT_ARRAY/test.c

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-only OR MIT
+/*
+
+Copyright (C) 2023 The Falco Authors.
+
+This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+or GPL2.txt for full copies of the license.
+
+*/
+
+/*
+ * Check that mm_struct's field `rss_stat` is an array.
+ * See 6.2 kernel commit: https://github.com/torvalds/linux/commit/f1a7941243c102a44e8847e3b94ff4ff3ec56f25
+ */
+
+#include "../../quirks.h"
+
+#include <uapi/linux/bpf.h>
+
+#include "../../ppm_events_public.h"
+#include "../../bpf_helpers.h"
+#include "../../types.h"
+#include "../../maps.h"
+
+// struct mm_struct declaration
+#include <linux/mm_types.h>
+
+#ifdef BPF_SUPPORTS_RAW_TRACEPOINTS
+#define BPF_PROBE(prefix, event, type)			\
```

Comment:
If we moved the macro definition in some header files (and note in `probe.c`) perhaps we might reuse the same definition here.
`types.h` might be a good candidate.
We can do it later anyway.